clinica/cmdline.py

```diff
@@ -62,7 +71,7 @@ def setup_logging(verbose: bool = False) -> None:
         {
             "logging": {"log_directory": os.getcwd(), "log_to_file": True},
```

Comment:
In my opinion, this should be the only line that requires a modification to support this use case: i.e. passing the `log_directory` by optional argument, still using `os.getcwd()` as default to keep compatibility with old behaviour).

You'd still need to mount an appropriate writable volume to your container to write the log files into and persist them after the container has finished running.